### DISCUSSION TITLE
Default (new) window/canvas size

### DISCUSSION THREAD

**Discussion by mibeon** [PintaProject/Pinta#994] [Q&A]
Hey folks,

Pinta starts with the default size of 800x600 pixels.
For my (fast) sketch needs is that to small. I am searching for the possibility to set the default canvas size, if I start fresh Pinta instance.

I'm using Ubuntu and the snap version of Pinta.
I found the configuration file here.
`/home/mb/snap/pinta/current/.config/Pinta/settings.xml`

Here are some settings, which take effect, but there is no setting for configuration of default canvas size.

Does anybody know how to set this?

Thanks for any help!

**Comment by cameronwhite** [comment]
Pinta should save the last image size you created in that settings file - do you see an entry like this?
```
<setting name="new-image-width" type="System.Int32">640</setting>
<setting name="new-image-height" type="System.Int32">480</setting>
```

That affects the initial image size when you open the New Image dialog.

But, I think the default blank image when you open a new instance of Pinta is always 800x600. I think it could be reasonable to switch that to follow the same preference though

**Reply by arseniiv** [reply]
Or probably a checkbox “Use this size for the default image” (or something) in that dialog.
